app/navigation/JourneyNavigator.scala

```diff
@@ -44,17 +46,23 @@ class JourneyNavigator @Inject()() extends Navigator {
 
     // Income from property, pensions and investments
     case RentalIncomePage                    => ua => controllers.propertypensionsinvestments.routes.PensionsController.onPageLoad(NormalMode, ua.taxYear)
-    case PensionsPage                        => ua => controllers.propertypensionsinvestments.routes.UkInsuranceGainsController.onPageLoad(NormalMode, ua.taxYear)
+    case PensionsPage                        => ua => controllers.propertypensionsinvestments.routes.UkInsuranceGainsController.onPageLoad(NormalMode,
```

Comment:
I assume the new line for ` ua.taxYear)` is done because of the formatting? If so, can we move the onPageLoad() method to the new line instead? what do you think?